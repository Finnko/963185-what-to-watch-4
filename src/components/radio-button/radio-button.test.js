import React from 'react';
import renderer from 'react-test-renderer';
import RadioButton from './radio-button.jsx';

const props = {
  type: `radio`,
  groupName: `rating`,
  id: `1`,
  value: `5`,
  checked: true,
  label: `rating 5`,
  onRadioChange: () => {},
};

describe(`RadioButton component render correctly`, () => {
  it(`Should RadioButton component render correctly`, () => {
    const tree = renderer
      .create(
          <RadioButton {...props}/>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
