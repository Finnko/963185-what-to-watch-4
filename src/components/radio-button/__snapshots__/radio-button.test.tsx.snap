// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton component render correctly Should RadioButton component render correctly 1`] = `
Array [
  <input
    checked={true}
    className="rating__input"
    disabled={false}
    id="1"
    name="rating"
    onChange={[Function]}
    type="radio"
    value="5"
  />,
  <label
    className="rating__label"
    htmlFor="1"
  >
    rating 5
  </label>,
]
`;
